### ISSUE TITLE
Do geometric.internal.MultiDihedral constraints work?

### ISSUE THREAD

**Issue by jminuse** [leeping/geomeTRIC#98]
I noticed code for MultiDihedral restraints in geometric.internal. As I read it, this is a single-degree-of-freedom restraint for a rotatable bond, constraining the mean of the torsion angles. I can add geometric.internal.MultiDihedral restraints to a coordinate system, but I am a bit wary because this type is not included in geometric.optimize.ParseConstraints() or in any examples I could find. Is this constraint type currently supported? Thanks!

**Comment by leeping** [comment]
Hello James,

I think this internal coordinate should work.  I did not make it part of the constraint interface or standard IC system, partly because I did not find any significant performance improvements.  The last time I used this was around three years ago, so "proceed with caution" is my advice. :)

The codes at internal.py:2146 show how one might set up an IC system using these degrees of freedom.  I mainly intended them for systems containing many "lines of atoms" such as the one in the attached image.

- Lee-Ping

<img width="182" alt="Screenshot 2019-09-25 15 25 05" src="https://user-images.githubusercontent.com/1441560/65644286-b5da6780-dfa8-11e9-9619-bc237cab44e9.png">
